paper/paper.md

```diff
@@ -0,0 +1,122 @@
+
+Executing AtlasReader on an MRI image will lead to the creation of the following
+four kinds of outputs:
+
+1. An **overview figure** showing the ROIs throughout the whole brain at once
+   (see Figure 1).
+2. For each ROI, an **informative figure** showing the sagittal, coronal and
+   transversal plane centered on the main peak of the ROI (see Figure 2).
+3. A **table** containing relevant information about the main **peaks** in each
+   ROI (see Figure 3).
+4. A **table** containing relevant information about the **cluster extent** of
+   each ROI (see Figure 4).
+
+Users have many parameters available to guide the creation of these outputs. For
+example, with `cluster_extent` a user can specify the minimum number of
+contiguous voxels required for a ROI to be shown in the output, `min_dstinace`
+can be used to extract peak information of more than just one peak within a
+given ROI, and `atlas` can be used to specify which atlases should be used for
+the output creation. By default, AtlasReader uses the AAL, the Desikan-Killiany,
+and the Harvard-Oxford atlases (see Figure 5). In the current version, users
+have also access to the Aicha, the Destrieux, the Juelich, the Marsatlas, the
+Neuromorphometrics, and the Talairach atlas.
+
+AtlasReader is licensed under the BSD-3 license and depends on the following
+python libraries: `matplotlib` [@matplotlib], `nibabel` [@nibabel],
+`nilearn` [@nilearn], `numpy` [@numpy], `scipy` [@scipy],
+`scikitlearn` [@scikitlearn] and `scikitimage` [@scikitimage].
+
+For a more detailed explanation about how AtlasReader works and instructions on
+how to install the software on your system, see https://github.com/miykael/atlasreader.
+
+
+# Example Output
+
+![Overview figure showing the ROIs throughout the whole brain at once.](fig_overview_figure.png){width=660}
+![Eight cluster figures, each centered on the main peak of the ROI, showing the sagittal, coronal and transversal plane of the ROI.](fig_cluster_figure.png){width=660}
+![Table showing relevant information for the main peaks of each ROI. This table contains the cluster association and location of each peak, its signal value at this location, the cluster extent (in mm, not in number of voxels), as well as the membership of each peak, given a particular atlas.](table_peak.png){width=100%}
+![Table showing relevant information for the cluster extent of each ROI. This table contains the cluster association and location of each peak, the mean value within the cluster, the cluster extent (in mm, not in number of voxels), as well as the membership of each cluster, given a particular atlas.](table_cluster.png){width=100%}
+![Individually colored label of the three default atlases, AAL, Desikan-Killiany and Harvard-Oxford, overlaid on the ICBM 2009c nonlinear asymmetric atlas. The Harvard-Oxford atlas is visualized differently because it is a probability atlas and therefore has overlapping regions.](fig_default_atlases.png){width=660}
```

Comment:
You reference "Figure [1-4]" in your summary above but there are no Figure #s here for any of the images -- should they be added?

Reply:
I was confused about this as well, but as much as I got from other JOSS publications, this will be added from JOSS itself. We will see. During submission you can generate the paper, see how it looks like and submit changes if you like.